code/game/objects/items/clown_items.dm

```diff
@@ -209,7 +209,10 @@
 	desc = "Damn son, where'd you find this?"
 	icon_state = "air_horn"
 	worn_icon_state = "horn_air"
-	soundfile = 'sound/items/airhorn2.ogg'
+
+/obj/item/bikehorn/airhorn/Initialize()
+	. = ..()
+	LoadComponent(/datum/component/squeak, list('sound/items/airhorn2.ogg'=1), 50, 0, 0, 0, 0, 20, 0)
```

Comment:
I don't think this will work, since it'll see the component exists then reuse that.

Making the component should just be in a separate proc that the airhorn overrides. i.e. `/obj/item/bikehorn/proc/make_squeak_component()`.

Reply:
uses the soundfile proc i did before now